src/main/java/org/sagebionetworks/bridge/udd/config/SpringConfig.java

```diff
@@ -13,58 +15,82 @@
-import com.amazonaws.services.sqs.AmazonSQSClient;
 import com.stormpath.sdk.api.ApiKey;
 import com.stormpath.sdk.api.ApiKeys;
 import com.stormpath.sdk.client.Client;
 import com.stormpath.sdk.client.Clients;
 import org.sagebionetworks.client.SynapseAdminClientImpl;
 import org.sagebionetworks.client.SynapseClient;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
-import org.sagebionetworks.bridge.config.Config;
-import org.sagebionetworks.bridge.config.PropertiesConfig;
 import org.sagebionetworks.bridge.crypto.AesGcmEncryptor;
 import org.sagebionetworks.bridge.crypto.Encryptor;
 import org.sagebionetworks.bridge.dynamodb.DynamoQueryHelper;
 import org.sagebionetworks.bridge.file.FileHelper;
 import org.sagebionetworks.bridge.heartbeat.HeartbeatLogger;
 import org.sagebionetworks.bridge.s3.S3Helper;
-import org.sagebionetworks.bridge.sqs.PollSqsWorker;
-import org.sagebionetworks.bridge.sqs.SqsHelper;
-import org.sagebionetworks.bridge.udd.worker.BridgeUddSqsCallback;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
+
+import javax.annotation.PostConstruct;
 
 // These configs get credentials from the default credential chain. For developer desktops, this is ~/.aws/credentials.
 // For EC2 instances, this happens transparently.
 // See http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/credentials.html and
 // http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/java-dg-setup.html#set-up-creds for more info.
 @ComponentScan("org.sagebionetworks.bridge.udd")
-@Configuration
+@Configuration("uddConfig")
 public class SpringConfig {
+    private static final String CONFIG_FILE = "BridgeUserDataDownloadService.conf";
+    private static final String DEFAULT_CONFIG_FILE = CONFIG_FILE;
+    private static final String USER_CONFIG_FILE = System.getProperty("user.home") + "/" + CONFIG_FILE;
+
+    private Properties envConfig;
+    private String ddbPrefix;
+
+    @PostConstruct
+    public void bridgeConfig() {
+        // setup conf file to load attributes
+        Path localConfigPath = Paths.get(USER_CONFIG_FILE);
+
+        Resource resource = new ClassPathResource(DEFAULT_CONFIG_FILE);
+
+        try {
+            envConfig = PropertiesLoaderUtils.loadProperties(resource);
+
+            if (Files.exists(localConfigPath)) {
+                Properties localProps = new Properties();
+                localProps.load(Files.newBufferedReader(localConfigPath, StandardCharsets.UTF_8));
+                envConfig = localProps;
+            }
+
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new RuntimeException(e);
+        }
```

Comment:
Spring beans can throw exceptions. Instead of catching the exception and re-throwing it, just declare that this method throws that exception.

Also, instead of writing the error to stderr, use a logger instead.